WoWsShipBuilder.ViewModels/Other/ShipComparisonDataSectionExtensions.cs

```diff
@@ -0,0 +1,64 @@
+﻿using WoWsShipBuilder.Core.Localization;
+
+namespace WoWsShipBuilder.ViewModels.Other;
+
+public static class ShipComparisonDataSectionExtensions
+{
+    public static string ShipComparisonDataSectionToString(this ShipComparisonDataSections dataSection)
```

Comment:
Shorten the name, use a name that implies it is not the normal string but is clear to understand, maybe something like ".AsString()"